iit/model_pairs/base_model_pair.py

```diff
@@ -374,12 +374,16 @@ def _check_early_stop_condition(self, test_metrics: MetricStoreCollection) -> bo
         """
         Returns True if all types of accuracy metrics reach 100%
         """
+        early_stop_accuracy_threshold = 99.5
```

Comment:
We should add this as a default training arg in `iit_model_pair` instead of putting it here.